omnichain/staking/contracts/Staking.sol

```diff
@@ -9,15 +9,17 @@ import "@zetachain/toolkit/contracts/BytesHelperLib.sol";
 contract Staking is ERC20, zContract {
     error SenderNotSystemContract();
     error WrongChain();
-    error NotAuthorizedToClaim();
+    error UnknownAction();
 
     SystemContract public immutable systemContract;
     uint256 public immutable chainID;
+    uint256 public rewardRate = 1;
+    uint256 constant BITCOIN = 18332;
```

Comment:
I think it would be more visual grouping immutable and constant together, no public between

Reply:
13e3680

Reply:
3f8585e